File: company-service/src/main/java/ru/panic/companyservice/util/SymbolsGeneratorUtil.java
package ru.panic.companyservice.util;

import java.security.SecureRandom;

public class SymbolsGeneratorUtil {
    private static final String CHARACTERS = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789%^&!@()-+=*#$";

    public static String generateRandomSymbols() {
        SecureRandom random = new SecureRandom();
        StringBuilder sb = new StringBuilder();

        for (int i = 0; i < 5; i++) {
            int randomIndex = random.nextInt(CHARACTERS.length());
            char randomChar = CHARACTERS.charAt(randomIndex);
            sb.append(randomChar);
        }

        return sb.toString();
    }
}

File: company-service/src/main/kotlin/ru/panic/companyservice/util/SymbolsGeneratorUtil.kt
package ru.panic.companyservice.util

import java.security.SecureRandom


object SymbolsGeneratorUtil {
    private const val CHARACTERS = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789%^&!@()-+=*#$"
    fun generateRandomSymbols(): String {
        val random = SecureRandom()
        val sb = StringBuilder()
        for (i in 0..5) {
            val randomIndex = random.nextInt(CHARACTERS.length)
            val randomChar = CHARACTERS[randomIndex]
            sb.append(randomChar)
        }
        return sb.toString()
    }
}
